<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyncWindow</class>
 <widget class="QWidget" name="SyncWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">SyncWindow{
	background-color: #F7F4F8;
}</string>
  </property>
  <widget class="QLineEdit" name="le_nameInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Event Name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_selectHours">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>147</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="maximum" stdset="0">
    <number>24</number>
   </property>
   <property name="minimum" stdset="0">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sync">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sync</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose a friend:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblIcon">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources.qrc">:/images/images/link.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
